integ-tests/src/main/java/com/google/avro/AvroHelper.java

```diff
@@ -16,31 +16,32 @@
 package com.google.avro;
 
 import static com.google.base.TestBase.TRAILING_SPACES_REGEX;
+import static java.lang.Double.parseDouble;
+import static java.lang.Integer.parseInt;
+import static java.lang.Long.parseLong;
 
 import java.io.File;
 import java.io.IOException;
 import java.nio.ByteBuffer;
 import org.apache.avro.file.DataFileReader;
 import org.apache.avro.generic.GenericDatumReader;
 import org.apache.avro.generic.GenericRecord;
-import org.apache.avro.io.DatumReader;
 
 /**
  * A helper class for reading and extracting data from Avro files.
  */
 public final class AvroHelper {
 
-  private AvroHelper() {}
+  private AvroHelper() {
```

Comment:
We should use google formatter and I'm pretty sure google formatter will write an empty code block on a single line.

Reply:
I believe, I used it in this class for formatting, and this is the result.
If you want, I can demo you this plugin configuration in my Idea.

Reply:
Done.
Indeed, for some reason the plugin didn't work after the IDE update.